yoti/yoti.admin.inc

```diff
@@ -24,8 +24,9 @@ function yoti_api_keys_settings($form, &$form_state) {
 
   $form['some_text'] = [
     '#markup' => '<div>' .
-    t('You need to first create a Yoti App at <a href="@yoti-dev" target="_blank">@yoti-dev</a>.', ['@yoti-dev' => YotiClient::DASHBOARD_URL]) . '</br >' .
-    t('Note: On the Yoti Dashboard the callback URL should be set to:') . ' <strong>' . url('/yoti/link', ['absolute' => TRUE]) . '</strong>' .
+    t('You need to first create a Yoti App at <a href="@yoti-dev" target="_blank">@yoti-dev</a>.', ['@yoti-dev' => YotiClient::DASHBOARD_URL]) . '<br>' .
+    t('Note: On the Yoti Dashboard the callback URL should be set to:') . ' <strong>' . url('/yoti/link', ['absolute' => TRUE]) . '</strong><br>' .
```

Comment:
Should we mention scenario here?
> On the Yoti Dashboard the callback URL **for the scenario** should be set to

Reply:
That's on my todo list and will a different ticket, this ticket is only for integrating ssl support in Docker.

Reply:
Do we currently have a ticket for updating the text in the plugins?